### PR TITLE
[Python] Exception handling for loading TF models

### PR DESCRIPTION
**Pull request by riteshghorse**
This PR adds exception handling for loading TF1 models. The current `TFModelHandlerTensor` and `TFModelHandlerNumpy` supports loading of TF2 models from the Tensorflow Hub. Thus adding clear exception message in case of TF1 models.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by riteshghorse** [comment]
assign set of reviewers

**Review by jrmccluskey — APPROVED** [review]
LGTM, the failed unit test workflows are re-running now

**Comment by riteshghorse** [comment]
Looks like Python 3.7 support is going away in 2.49. So we can ignore the 3.7 failing tests https://github.com/apache/beam/pull/27194

**Comment by jrmccluskey** [comment]
Those are also broken because of a flawed dependency update, I'll get the issue number to link it in here shortly

**Comment by jrmccluskey** [comment]
#27193 for reference

File: sdks/python/apache_beam/ml/inference/tensorflow_inference.py [hunk 1]
```diff
@@ -56,7 +56,15 @@ class ModelType(enum.Enum):
...
-  model = tf.keras.models.load_model(hub.resolve(model_uri), **load_model_args)
+  try:
+    model = tf.keras.models.load_model(
+        hub.resolve(model_uri), **load_model_args)
+  except Exception as e:
+    raise ValueError(
+        "Unable to load the TensorFlow model: {exception}. Make sure you've \
+        saved the model with TF2 format. Check out the list of TF2 Models on \
+        TensorFlow Hub - https://tfhub.dev/s?subtype=module,placeholder&tf-version=tf2."  # pylint: disable=line-too-long
+        .format(exception=e))
```
[the remark this answers is not in the document]
Comment (AnandInguva) [reply sdks/python/apache_beam/ml/inference/tensorflow_inference.py:67]: Ah this is merged already. NVM then.

**Pull request merged**